.github/workflows/nodejs.yml

```
@@ -0,0 +1,75 @@
+name: s3rver
```

Not a huge deal. But I tend to suggest that naming is more specific for GitHub Actions workflows. If you ever want to add workflows to close stale issues, tag PRs, publish new versions, etc. those will tend to be different workflows. Which would make this name confusing since it's so general.

For [Dynamoose](https://github.com/dynamoose/dynamoose/blob/master/.github/workflows/ci.yml#L1) we use `CI`. Not sure that makes the most amount of sense tho.

```suggestion
name: CI
```

---

I also think it'd be valuable to have the name match the file name.

---

Super nit-picky, but I thought it was worth at least mentioning.

---

CI makes enough sense to me, and thank you for giving me a new pet peeve for when I'm looking at other projects' actions 😄.